Code/GraphMol/Descriptors/Wrap/testMolDescriptors.py

```diff
@@ -41,6 +41,32 @@ def testAtomPairTypes(self):
     self.assertTrue(rdMD.GetAtomPairAtomCode(mol.GetAtomWithIdx(1),2)==\
                     0 | (2 | 1<<params.numPiBits)<<params.numBranchBits)
 
+  def testAtomPairTypesChirality(self):
+    mols = [Chem.MolFromSmiles(x) for x in ("CC(F)Cl","C[C@@H](F)Cl","C[C@H](F)Cl")]
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)),
+        rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1)))
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1)))
+    self.assertEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1)))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1),includeChirality=True))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[0].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1),includeChirality=True))
+    self.assertNotEqual(rdMD.GetAtomPairAtomCode(mols[1].GetAtomWithIdx(1),includeChirality=True),
+        rdMD.GetAtomPairAtomCode(mols[2].GetAtomWithIdx(1),includeChirality=True))
```

Comment:
Yeah, I agree with Greg with respect for testing.  I would have done:

```
 mol1,mol2,mol3 = [Chem.MolFromSmiles(x) for x in ("CC(F)Cl","C[C@@H](F)Cl","C[C@H](F)Cl")]

for m1, m2 in [(mol1, mol2), (mol1, mol3)]:
```

myself :)